client-src/elements/chromedash-feature-page.js

```diff
@@ -175,11 +175,20 @@ export class ChromedashFeaturePage extends LitElement {
 
     window.csClient.getFeatureLinks(this.featureId).then(
       (featureLinks) => {
-        this.featureLinks = featureLinks;
+        this.featureLinks = featureLinks?.data || [];
+        if (featureLinks?.has_stale_links) {
+          // delay 2 seconds to fetch server to get latest link information
+          setTimeout(this.refetchFeatureLinks.bind(this), 2000);
+        }
       },
     );
   }
 
+  async refetchFeatureLinks() {
+    featureLinks = await window.csClient.getFeatureLinks(this.featureId, false);
```

Comment:
@jrobbins is below changes what you prefer?

```suggestion
    const updateStaleLinks = false
    featureLinks = await window.csClient.getFeatureLinks(this.featureId, updateStaleLinks);
```

Reply:
My mistake.  I was thinking of python code.   What you had originally is fine for javascript.